### PR TITLE
Cover Card Updates

### PR DESCRIPTION
**Pull request by benbur98**
I have updated the Cover Card with some features/enhancements as requested in Issues #818 , #344 and #877 .

This involves:
- Inverting the percentage for soe covers with a variable to set this - also update the Open/Closed state label.
- Including an additional option for choosing either large/small garage icons
- Adding Tilt control buttons as an additional line in the expanded card view

Unfortunately I don't have any covers (and they're not a Virtual Component), so I haven't been able to test this. If anyone from these issues would be able to test it then that would be great.

### CODE REVIEW

**Comment by stelbo** [comment]
Hi, I tried to test it but the card did not change and still looks same. 

Here is my config in ui-lovelcace.yaml: 

              - type: "custom:button-card"
                template:
                  - card_cover
                  - cover
                entity: cover.obyvacka_dvere_2
                variables:
                  ulm_card_cover_enable_controls: true
                  ulm_card_cover_enable_slider: true
                  ulm_card_cover_enable_popup: true
                  ulm_card_cover_enable_tilt: true

**Comment by benbur98** [comment]
@stelbo thanks for testing this for me! Just to check have you modified the code for this component locally or added it as a new custom component for testing? The PR is not merged so the card won't work in this new way yet.

**Review by basbruss — APPROVED** [review]
I have adjusted and improved some logic you have made to the cards.
Thanks for the hard work and time 😄

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 1]
```diff
@@ -489,3 +510,175 @@ card_media_player:
...
+                      template: "popup_media_player_infos"
+                      entity: "[[[ return entity.entity_id; ]]]"
+              tap_action:
+                action: "call-service"
+                service: "media_player.volume_set"
+                service_data:
+                  entity_id: "[[[ return variables.ulm_card_media_player_player_controls_entity ]]]"
+                  volume_level: |
+                    [[[
+                      var volume = states[entity.entity_id].attributes.volume_level;
```
[the remark this answers is not in the document]
Comment (benbur98) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:651]: Ah maybe, I'd have to look into it more. 
This PR is already closed and merged so would need a new PR if you would like to make the changes.

**Pull request merged**